tests/cxx/iwyu_stricter_than_cpp.cc

```diff
@@ -139,13 +139,19 @@ void TestTypeAliases() {
 }
 
 void TestAutocast() {
-  // We need full type of is2 because the declarer of Fn didn't
+  // We need full type of ic2 because the declarer of FnValues and FnRefs
+  // didn't.
   // IWYU: IndirectStruct2 is...*iwyu_stricter_than_cpp-i2.h
-  Fn(1, 2, 3, 4, 5);
+  FnValues(1, 2, 3, 4, 5);
+  // IWYU: IndirectStruct2 is...*iwyu_stricter_than_cpp-i2.h
+  FnRefs(1, 2, 3, 4, 5);
 
-  // We need full type of is2 because the declarer of Fn didn't
+  // We need full type of ic2 because the declarer of TplFnValues and TplFnRefs
```

Comment:
Let's spell out `IndirectStruct2` instead of `ic2`

Reply:
Overlooked, thanks!